CONTRIBUTING.adoc

```diff
@@ -103,11 +103,12 @@ explains where the component fits in the architecture, and anything else that yo
 
 === Logs
 
-We use SLF4J; loggers are declared like this:
+We use jboss logger; loggers are declared like this:
```

Comment:
So i'm hesitating. According to [this](https://quarkus.io/guides/logging) and [this](https://antoniogoncalves.org/2019/06/07/configuring-a-quarkus-application/) we have many choices, and SLF4J is supported.

Reply:
When analyzing other extensions in the quarkus repository you will notice that all of them are using JBoss instead of slf4j

Reply:
But wouldn't it be safer to just keep the current code? 

The only thing I was planning to add is dependency declarations in all modules using SLF4J:

```
      <dependency>
        <groupId>org.slf4j</groupId>
        <artifactId>slf4j-api</artifactId>
      </dependency>
      <dependency>
        <groupId>org.jboss.logging</groupId>
        <artifactId>jboss-logging</artifactId>
        <scope>runtime</scope>
      </dependency>
```

Reply:
Other extensions are using jboss logger API (i.e `debugf` instead of `debug`)

Reply:
- sl4j is already excluded from java-driver in the quarkus - I've excluded the slf4j form java-driver in the `quickstart`
- I am not sure if we need to add the dependency to JBoss explicitly. It is already in the quarkus dependency
wdyt?

Reply:
> it is usually considered good manners to always declare dependencies that are directly used by the module.

Although I confess that the Quarkus framework doesn't seem to follow this pattern, so I'm OK with not adding the dependency explciitly.